README.md

```diff
@@ -9,12 +9,45 @@ Example provider
 
 1. Create a token at https://lichess.org/account/oauth/token/create?scopes[]=engine:read&scopes[]=engine:write
 
-2. Run:
+2. Install your favorite engine. 
+   (Under Windows is it is recommended to place it directly in ```C```. Under Unix choose ```usr/bin/YOUR_ENGINE_FOLDER/program```)
```

Comment:
Not sure I'd recommend either of those. On Unix, it should be `/usr/bin/engine-name` or any other directory in the PATH. On Windows, I guess it's harder to say but I'd at least create a directory to put the engines in if you put them directly into `C`. Also, you can just use single backticks for inline code.